docs/_snippets/addon-a11y-parameter-remove.md

```diff
@@ -0,0 +1,43 @@
+```ts filename="Button.stories.ts" renderer="common" language="ts"
+// Replace your-renderer with the renderer you are using (e.g., react, vue3)
+import { Meta } from '@storybook/your-renderer';
+
+import { Button } from './Button';
+
+const meta: Meta<typeof Button> = {
+  component: Button,
+  parameters: {
+    // 👇 Remove this once all stories pass accessibility tests
+    // a11y: { test: 'error' },
```

Comment:
Can you adjust this to the `todo` instead of `error` when you're able? This is primarily to align with the documentation flow and the placement of the snippet.

Reply:
Oh, great catch! That would've been confusing.